app/models/paper.rb

```diff
@@ -43,4 +45,8 @@ def valid_methodologies
       'case report'
     ]
   end
+
+  def find_publication
+    self.publication = Publication.where(name: self.publication.name).first_or_initialize
```

Comment:
Before you fix it, do you use this anywhere?

On Tue, Aug 23, 2016, 12:12 PM wsf246 <notifications@github.com> wrote:

> In app/models/paper.rb
> <https://github.com/briankung/projectcredo/pull/64#discussion_r75899350>:
>
> > @@ -43,4 +45,8 @@ def valid_methodologies
> >        'case report'
> >      ]
> >    end
> > +
> > +  def find_publication
> > +    self.publication = Publication.where(name: self.publication.name).first_or_initialize
>
> it breaks, fixing it now.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/briankung/projectcredo/pull/64/files/2b8cccb48a7d372bd7c56a40f6732f85f108ff4b#r75899350>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACtGx5YVRK3gIEawX89XTkiZ4b9PDwnRks5qixvcgaJpZM4JqR56>
> .
>